modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -174,28 +248,45 @@ definition {
 				thirdInvoice = "Document_1.png");
 		}
 
-		task ("And I go back to the MDF listing page") {
-			PRMUtils.gotoPRM();
+		task ("And given that the Manager user is logged in and goes to MDF listing page") {
```

Comment:
Hi @marinaasant0s, it should be `'...Claim listing page'` instead of `'...MDF listing page'`